ShowMyData/Models/DrugCrime.cs

```diff
@@ -17,14 +17,18 @@ namespace Drugs
 
     public partial class DrugCrime
     {
+        private string addressX;
+        private string latitudeX;
+        private string longitudeX;
+
         [JsonProperty("address_x")]
-        public string AddressX { get; set; }
```

Comment:
Encapsulation is a good idea, though {get; set; } syntax is often considered a shortcut for encapsulation.  In our class, I'm fine with either approach.